pastepwn/actions/sendemailaction.py

```diff
@@ -0,0 +1,33 @@
+import smtplib
+import re
+from .basicaction import BasicAction
+
+class SendEmailAction(BasicAction):
+    """Action to send a Email containing the paste through SMTP protocol to a certain email-id"""
+    name = "SendEmailAction"
+
+    def __init__(self, username,password,reciever_email,hostname,port):
+        super().__init__()
+
+        if not re.match(r"^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$", username) or username is None:
+            raise ValueError("username not correct or None!")
+        if not re.match(r"^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$", reciever_email) or reciever_email is None:
+            raise ValueError("reciever_email not correct or None!")
+        self.username = username
+        self.password=password
+        self.reciever_email=reciever_email
+        self.hostname="smtp."+hostname+".com"
```

Comment:
This prevents domains apart from `.com` domains. Better would be to have the user insert the exact smtp domain themself => `self.hostname = hostname`